mis_builder/models/mis_report_style.py

```diff
@@ -163,16 +164,48 @@ def merge(self, styles):
         return r
 
     @api.model
-    def render(self, lang, style_props, type, value):
+    def render(self, lang, style_props, type, value, currency=None):
         if type == 'num':
             return self.render_num(lang, value, style_props.divider,
                                    style_props.dp,
                                    style_props.prefix, style_props.suffix)
         elif type == 'pct':
             return self.render_pct(lang, value, style_props.dp)
+        elif type == 'mon':
+            return self.render_mon(lang, value, style_props.prefix,
+                                   style_props.suffix, currency)
         else:
             return self.render_str(lang, value)
 
+    @api.model
+    def render_mon(self, lang, value, prefix=None, suffix=None,
+                   currency=None):
+        # format number following user language
+        if value is None:
+            return u''
+        elif value is AccountingNone:
+            value = 0.0
```

Comment:
AccountingNone must render as an empty string.

Reply:
Why did I changed that .... probably a good reason but I don't remember..

Reply:
I think that we've had clients who prefered to have 0.00 EUR instead of a empty cell in their reports

Reply:
MIS Builder goes to great lenght at distinguishing 0 from no data.

For situations where customers insist at having 0.0 displayed for no data, it can be handled in expressions like this: `balp[...] or 0.0`.